Src/AutoFixture.NUnit3.UnitTest/FrozenAttributeIntegrationTest.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using NUnit.Framework;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    [TestFixture(Description="This test suite is not exhaustive")]
+    public class FrozenAttributeIntegrationTest
```

Comment:
Isn't this a [Facade Test](http://blog.ploeh.dk/2012/06/27/FacadeTest) rather than an Integration Test?